corehq/apps/es/users.py

```diff
@@ -63,13 +63,18 @@ def show_only_inactive(self):
         return query.is_active(False)
 
 
-def domain(domain, allow_mirroring=False):
+def domain(domain, allow_mirroring=False, include_mirrors=False):
```

Comment:
this could use a docstring to explain what these two `mirroring` args mean

Reply:
the new param was removed later